### DISCUSSION TITLE
Closing Apps Using Flow Launcher

### DISCUSSION THREAD

**Discussion by osama-almangathi** [Flow-Launcher/Flow.Launcher#1503] [Q&A] [answered]
How to close an application using flow launcher?

**Comment by VictoriousRaptor** [comment, marked as answer]
You can use the integrated process killer plugin. Action word "kill" by default. Sorry for replying so late.

**Reply by osama-almangathi** [reply]
I guess kill is used for something else, for killing a process I think!. I really don't understand this, but anyway what I meant is how to close a window, any window whether it is for an app or not. I'm just tired from clicking the X button every time I need to close a window. It's much easier to do it with keyboard. I hope you got my point!

**Reply by jjw24** [reply]
You can also try the WindowWalker plugin, select the window result and press Ctrl+ Enter on it, this will close the window. I am not sure however if it's a safe close or process kill so you need to just double check that.
